app/components/graphs/ResaleValueLineChart.tsx

```diff
@@ -57,39 +52,75 @@ const ResaleValueLineChart: React.FC<ResaleValueLineChartProps> = ({
   selectedMaintenance,
   maxY
 }) => {
+  const [hoveredLine, setHoveredLine] = useState<string | null>(null); // We use useState to figure out what line is being hovered over for the tooltip
+
   const renderLine = (dataKey: keyof Omit<DataPoint, "year">) => (
     <Line
       type="monotone"
       dataKey={dataKey}
-      stroke={`var(--color-${dataKey})`}
+      stroke={dataKey === selectedMaintenance 
+        ? `rgb(var(--fairhold-equity-color-rgb))` 
+        : `rgb(var(--fairhold-interest-color-rgb))`}
       strokeWidth={2}
-      strokeDasharray={dataKey === selectedMaintenance ? "0" : "5 5"}
       dot={false}
-    />
+      activeDot={false}
+      onMouseOver={() => setHoveredLine(dataKey)}
+      onMouseOut={() => setHoveredLine(null)}
+      >
+      <LabelList
+        content={({ x, y, index }) => {
```

Comment:
This is quite a lot of complexity for an inline prop, could we extract this out?

```suggestion
        content={LabelListContent}
```

Reply:
That's a great shout! I had a quick go and got some type issues that I couldn't figure out quickly--I also realised that this is a pattern that's reused across `ResaleValueLineChart`, `HowMuchFHCostBarChart`, `HowMuchPerMonthBarChart` and `HowMuchFHCostNotViableBarChart`, so I'm going to merge this as is and make a [new card](https://github.com/orgs/theopensystemslab/projects/5?pane=issue&itemId=109795058&issue=theopensystemslab%7Cfairhold-dashboard%7C483) to fix all instances of it in one go